src/main/java/transforms/ArithmeticTransform.java

```diff
@@ -0,0 +1,109 @@
+package transforms;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.openlattice.shuttle.dates.DateTimeHelper;
+import com.openlattice.shuttle.transformations.Transformation;
+import com.openlattice.shuttle.util.Constants;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+public class ArithmeticTransform extends Transformation<Map<String, String>> {
+    private String               operator;
+    private List<Transformation> leftTransforms, rightTransforms;
+    private              Double alternative;
+    private static final Logger logger = LoggerFactory.getLogger( DateTimeHelper.class );
+
+    /**
+     * Represents a transformation from string to datetime.
+     *
+     * @param leftTransforms:  transformations for left side of computation
+     * @param rightTransforms: transformations for right side of computation
+     * @param operator:        operator, one of +, -, /, *
+     * @param alternative:     if left or right transformation fail to parse to double: what should the value be (defaults to null);
+     */
+    public ArithmeticTransform(
+            @JsonProperty( Constants.LEFTTRANSFORMS ) List<Transformation> leftTransforms,
+            @JsonProperty( Constants.RIGHTTRANSFORMS ) List<Transformation> rightTransforms,
+            @JsonProperty( Constants.OPERATOR ) String operator,
+            @JsonProperty( Constants.ELSE ) Optional<Double> alternative
+    ) {
+        this.operator = operator;
+        this.leftTransforms = leftTransforms;
+        this.rightTransforms = rightTransforms;
+        this.alternative = alternative.orElse( null );
+    }
+
+    @JsonProperty( Constants.OPERATOR )
+    public String getOperator() {
+        return operator;
+    }
+
+    @JsonProperty( Constants.LEFTTRANSFORMS )
+    public List<Transformation> getLeftTransforms() {
+        return leftTransforms;
+    }
+
+    @JsonProperty( Constants.RIGHTTRANSFORMS )
+    public List<Transformation> getRightTransforms() {
+        return rightTransforms;
+    }
+
+    public Double getValueFromTransform( Object transformed ) {
+
+        if ( StringUtils.isBlank( String.valueOf( transformed ) ) ) {
+            logger.debug( "Unable to parse number {} for arithmetic transform, returning",
+                    String.valueOf( transformed ),
```

Comment:
String.valueOf() transforms `null` into `""`, which is the desired behavior here.

Reply:
I think it will actually transform `null` to `"null"` which should be the same as omitting `String.valueOf()`

Reply:
But in any case, since you check if `transformed` is blank above, you can probably just omit that argument alltogether

Reply:
Also StringUtils.isBlank can also operate directly on transformed I think (it correctly handles null as blank).

Reply:
You might also want to make sure object is instanceof String and cast at the very beginning instead of propagating String.valueOf(...) all the way down.

Reply:
Interesting ! I did not know that it would be `"null"` :-o 

This is done and added a test for null input.